deploy/olm-catalog/datadog-operator/0.1.1/datadog-operator.v0.1.1.clusterserviceversion.yaml

```diff
@@ -53,7 +77,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: datadog-operator
-                image: datadog/operator:latest
+                image: datadog/operator:0.1.1
```

Comment:
can we still use "latest" here?

Reply:
IMHO, I believe we should correlate CSV version and image version, we might be do code changes that require changes in the CSV. As OLM is meant to have auto-upgrade, I believe we should always bundle the release.

Reply:
agree, but for now we don't have this image released

Reply:
we should generate the OLM when we release the a new operator version

Reply:
We can stay with 0.1.1 and don't forget to deliver the next version with this value

Reply:
I do have pushed a 0.1.1 for my tests